README.md

```diff
@@ -289,26 +289,36 @@ The action function always uses two parameters: the `inputs` you defined in your
 The `context` object exposes useful methods to structure your action.
 
 - `context.log(...logs)` allows the user to log messages and make them available on Clay
-- `context.success({ data, textPreview, imagePreview })` generates a return object indicating a success for the action function
+- `context.success({ data, textPreview, imagePreview, successType: context.status.SUCCESS_TYPE })` generates a return object indicating a success for the action function
 - `context.fail({ message, errorType: context.status.ERROR_TYPE })` generates a return object indicating a failure of the action function
 
-#### Handling Errors
-The `context` object provides structured errors. This ensures that the Clay UI shows a proper error message for the users of your action.
+#### Handling Return Statuses
+The `context` object provides structured statuses. This ensures that the Clay UI shows a proper message for the users of your action.
 
-> Providing structured errors is extremely important to communicate with your Action's user. Without structured errors, the user won't know what went wrong.
+> Providing structured statuses is extremely important to communicate with your Action's user. 
+>
+> Sending a success status allows you to suggest more information about your action return data.
+> Sending an error status, can indicate to the user exactly what went wrong. 
 
-`context.status` contains status codes for most error types:
+`context.status` contains status codes for the following status types:
 
 ```
+context.status.SUCCESS_NO_DATA
 context.status.ERROR_MISSING_INPUT
 context.status.ERROR_INVALID_INPUT
 context.status.ERROR_MISSING_OUTPUT_DATA
 context.status.ERROR_INVALID_OUTPUT_DATA
 context.status.ERROR_BAD_REQUEST
 context.status.ERROR_TIMEOUT
 context.status.ERROR_INVALID_CREDENTIALS
+context.status.SUCCESS
+context.status.ERROR
+context.status.RETRY
```

Comment:
now that you have removed the different retry types from clay-action-client, can we get rid of context.status.retry here?